jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -2474,6 +2474,56 @@ ecma_object_class_is (ecma_object_t *object_p, /**< object */
   return false;
 } /* ecma_object_class_is */
 
+/**
+ * Checks if the given argument is an array or not.
+ *
+ * @return true - if the given argument is an array object
+ *         false - otherwise
+ */
+inline bool JERRY_ATTR_ALWAYS_INLINE
+ecma_object_is_array (ecma_value_t arg) /**< argument */
```

Comment:
I feel like this does not fit here very well. I think this would be better off as `ecma_is_value_array` in `ecma-helpers-value.c` .